scripts/yul_coverage.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -e
+
+ROOT_DIR="$(dirname "$0")"/..
+
+FAILED=()
+SUCCESS=()
+test_file()
+{
+  local SOL_FILE
+  local OUTPUT
+
+  SOL_FILE=${1}
+  OUTPUT=$(solc --ir "${SOL_FILE}" 2>&1 || true)
+  if [[ ${OUTPUT} == *"UnimplementedFeatureError"* ]]; then
+    FAILED+=("${SOL_FILE}")
+  else
+    SUCCESS+=("${SOL_FILE}")
+  fi
+}
+
+# we only want to use files that do not contain errors or multi-source files.
+SOL_FILES=()
+while IFS='' read -r line; do
+  SOL_FILES+=("$line")
+done < <(
+  grep -riL -E \
+    "^\/\/ (DocstringParsing|Syntax|Type|Parser|Declaration)Error|^==== Source:" \
+    "${ROOT_DIR}/test/libsolidity/syntaxTests" \
+    "${ROOT_DIR}/test/libsolidity/semanticTests"
+)
+
+echo "running tests..."
+for SOL_FILE in "${SOL_FILES[@]}"; do
+  test_file "${SOL_FILE}"
```

Comment:
You have 2-space indents and I see that nearly all other scripts use 4-spaces or tabs (or, unfortunately a mix of 4-spaces and tabs).

Reply:
will check